### DISCUSSION TITLE
Bun v1.1.30

### DISCUSSION THREAD

**Discussion by Jarred-Sumner** [oven-sh/bun#14408] [Announcements]
To install:
```bash
curl -fsSL https://bun.sh/install | bash
# or you can use npm
# npm install -g bun
```

Windows:
```bash
powershell -c "irm bun.sh/install.ps1|iex"
```

To upgrade to Bun v1.1.30:
```bash
bun upgrade
```

### Thanks to 17 contributors!

- [@190n](https://github.com/190n)
- [@cirospaciari](https://github.com/cirospaciari)
- [@dylan-conway](https://github.com/dylan-conway)
- [@Electroid](https://github.com/Electroid)
- [@Jarred-Sumner](https://github.com/Jarred-Sumner)
- [@KiwiZ0](https://github.com/KiwiZ0)
- [@lewismiddleton](https://github.com/lewismiddleton)
- [@matubu](https://github.com/matubu)
- [@metonym](https://github.com/metonym)
- [@mjomble](https://github.com/mjomble)
- [@nektro](https://github.com/nektro)
- [@paperdave](https://github.com/paperdave)
- [@robertshuford](https://github.com/robertshuford)
- [@snoglobe](https://github.com/snoglobe)
- [@versecafe](https://github.com/versecafe)
- [@wpaulino](https://github.com/wpaulino)
- [@Xmarmalade](https://github.com/Xmarmalade)
- [@zackradisic](https://github.com/zackradisic)

<hr /><em>This discussion was created from the release <a href='https://github.com/oven-sh/bun/releases/tag/bun-v1.1.30'>Bun v1.1.30</a>.</em>

**Comment by RobinTail** [comment]
```sh
brew upgrade bun
Warning: oven-sh/bun/bun 1.1.29 already installed
```

could you please push it to homebrew, @Jarred-Sumner ?

UPD: fixed

```
==> Upgrading oven-sh/bun/bun
  1.1.29 -> 1.1.30
```

**Comment by nurulhudaapon** [comment]
The link to [LightningCSS](https://github.com/devongovett/lightningcss) is broken on the blog.

**Reply by nurulhudaapon** [reply]
Expected link should be: https://github.com/parcel-bundler/lightningcss

**Comment by RobinTail** [comment]
Noticed this issue in 1.1.30:
https://github.com/oven-sh/bun/issues/14415

**Comment by j-joker** [comment]
Instead of adding these fancy features, why not focus on solving a hardcore problem, like supporting Next.js?

**Reply by Jarred-Sumner** [reply]
What issue are you running into? People are using Next with Bun. And have been for many months now. The experience isn’t great yet, we need to be faster. But we are working on that.

**Reply by j-joker** [reply]
<img width="634" alt="image" src="https://github.com/user-attachments/assets/655b3ff9-aa16-4f4b-ba12-e199ad9a67dd">

As the documentation from bun offcial site suggests, we are currently unable to utilize Bun as the runtime environment for Next.js to accelerate server-side rendering (SSR) performance.
I agree it needs to be faster. Given Next.js's massive user base, this would be an excellent opportunity to promote Bun.

**Reply by mavyfaby** [reply]
@j-joker I guess they're working on it
